esp/esdllib/esdl_def.cpp

```diff
@@ -977,7 +977,7 @@ static SecAccessFlags translateAuthLevel(const char* flag)
         return SecAccess_Full;
 
     DBGLOG("Unknown access level: %s", flag);
-    return SecAccess_Full;
+    throw( MakeStringException(0, "Unknown access level: %s", flag) );
```

Comment:
@afishbeck this seems fine and prob the correct approach as long as hidlcomp handles the exception correctly.
I think it was intended to set the restriction as high as possible by default, but forcing the developer to set the level correctly is better.